tasks/db_deploy/migrate_db.py

```diff
@@ -23,26 +23,30 @@ def perform_migration(current_schema_version: int, config: Dict[str, str]) -> No
     # Determine migrations to run based on current_schema_version and
     # update the versions table based on the latest_schema_version.
     if current_schema_version == 1:
-        # Run migrations from version 1 to the latest version
-        # in this case version 2 is the latest so we set the latest version
-        # flag to True
+        # Run migrations from version 1 to version 2
```

Comment:
if schema version == 1, you will need to run migrations from 1 - 2 and from 2 - 3.

Reply:
changed to 
```
    if current_schema_version == 1:
        # Run migrations from version 1 to version 2
        migrate_versions_1_to_2(config, False)
        current_schema_version = 2
        # Run migrations from version 2 to version 3
        migrate_versions_2_to_3(config, True)
```

Reply:
Review the changes andrew made in his PR to do migrations, it is a little more elegant way to handle them instead of repeating function calls over and over again. Possibly adopt that pattern instead.